dora/tests/integration/src/test/java/alluxio/client/fs/io/DoraPositionReaderIntegrationTest.java

```diff
@@ -76,6 +76,7 @@ public static Collection<Object[]> data() {
           .setProperty(PropertyKey.WORKER_PAGE_STORE_PAGE_SIZE, Constants.KB)
           .setProperty(PropertyKey.WORKER_PAGE_STORE_SIZES, "1GB")
           .setProperty(PropertyKey.USER_NETTY_DATA_TRANSMISSION_ENABLED, true)
+          .setProperty(PropertyKey.UNDERFS_XATTR_CHANGE_ENABLED, false)
```

Comment:
I propose to have a new patch for such change.
Don't mix irrelevant changes in the same patch.

Reply:
reverted